README.md

```diff
@@ -1 +1,185 @@
+>
+> <div>
+>
+  <img src="src/assets/img/comment.png" alt="comment-popup-window" width="500" height="300" />
+  <img src="src/assets/img/reservation.png" alt="reservation-popup-window" width="500"  height="300" />
+   <img src="src/assets/img/home.png" alt="home-page" width="500" height="300/>
+  <div>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+>Please follow the instructions to clone the repo
+To get a local copy, follow these simple example steps.
+Clone this repository or download the Zip folder:
+
+### Prerequisites
+
+>In order to check the linters errors make sure you have installed the [nodejs](https://nodejs.org)
+
+
+### Setup
+
+>Clone this repository to your desired folder: cd [folder] to navigate and run the below comand to clone the project
+
+```sh
+git clone https://github.com/raihan2bd/js-capstone.git
+```
+
+
+### Install
+
+Install this project with:
+
+download npm the packages
+```sh
+npm install --save-dev
+```
+
+### Usage
+
+ > To see the project in your local browser first of all, go to the dist folder then open the index.html file.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="authors"></a>
+
+
+
+👤 **Abu Raihan**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/raihan2bd)
+
+👤 **Salwa Ballouti**
+
+- GitHub: [@githubhandle](https://github.com/Salwa99)
+- Twitter: [@twitterhandle](https://twitter.com/salwa-ballouti)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/salwa-ballouti)
+
+👤 **Lugard Agu**
+
+- GitHub: [@githubhandle](https://github.com/lugard1)
+- Twitter: [@twitterhandle](https://twitter.com/Dsn3kings)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/lugard-agu-45bb05b6/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## How to access the database?
+
+- It's not available for now I'm still working on it, once it's available I'll update this answer.
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Give a ⭐️ if you like this project!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Without Microverse Help this project not compleated at all so thanks a lot Microverse for giving us this kind of opertunity.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>

```

Comment:
- [ ] **[Optional] Invalid MIT link 😨**
    I think it would be better to provide a  valid **LICENSE** link in your **README** as open source software needs to be licensed to provide rights to **modify**, **distribute** other than creator :joy: 

    _we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._